changelog.d/20230403_121625_samuel.guillaume_scrt_3269_ggshield_scan_removal.md

```diff
@@ -0,0 +1,40 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+### Removed
+
+- Removed the deprecated commands `ggshield scan` and `ggshield ignore`.
```

Comment:
When we remove commands, it's a good idea to remind the user of what to use instead. I suggest the following:
```suggestion
- The deprecated commands `ggshield scan` and `ggshield ignore` have been removed. Use `ggshield secret scan` and `ggshield secret ignore` instead.
```